src/translations/en.json

```diff
@@ -124,7 +126,7 @@
     "RatioTap3": "Ratio tap 3",
     "PhaseTap3": "Phase tap 3",
 
-    "viewOverloadsTable": "View overloads table",
+    "viewOverloadsTable": "View overload table: ",
```

Comment:
There are several overloads

Reply:
This change was made on purpose, it was required.